<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FreeEGWidgetClass</class>
 <widget class="QWidget" name="FreeEGWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1181</width>
    <height>529</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1181</width>
    <height>453</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FreeEGWidget</string>
  </property>
  <widget class="QGraphicsView" name="egView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1155</width>
     <height>259</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupTrack_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>261</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Track 1</string>
   </property>
   <widget class="ComboPicker" name="trackParam_1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="ComboPicker" name="trackSceneSw_1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>Off</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Scene 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Scene 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Both</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Parameter:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Scene Switch:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupTrack_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>320</y>
     <width>261</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Track 2</string>
   </property>
   <widget class="ComboPicker" name="trackParam_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Parameter:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Scene Switch:</string>
    </property>
   </widget>
   <widget class="ComboPicker" name="trackSceneSw_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>Off</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Scene 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Scene 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Both</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupTrack_4">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>320</y>
     <width>261</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Track 4</string>
   </property>
   <widget class="ComboPicker" name="trackParam_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Parameter:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Scene Switch:</string>
    </property>
   </widget>
   <widget class="ComboPicker" name="trackSceneSw_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>Off</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Scene 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Scene 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Both</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupTrack_3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>320</y>
     <width>261</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Track 3</string>
   </property>
   <widget class="ComboPicker" name="trackParam_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Parameter:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Scene Switch:</string>
    </property>
   </widget>
   <widget class="ComboPicker" name="trackSceneSw_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>Off</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Scene 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Scene 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Both</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QLabel" name="label_43">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>380</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Key Track</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="DialKnob" name="keyTrack">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>330</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="minimum">
    <number>-64</number>
   </property>
   <property name="maximum">
    <number>63</number>
   </property>
   <property name="notchTarget">
    <double>16.000000000000000</double>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>1151</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Current track edit:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="currentTrackCombo">
      <item>
       <property name="text">
        <string>Track 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Track 2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Track 3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Track 4</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Legnth:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="ComboPicker" name="lengthCombo">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Loop:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="ComboPicker" name="loopCombo">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <item>
       <property name="text">
        <string>Off</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Forward</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Forward Half</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Alternate</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Alternate Half</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Trigger:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="ComboPicker" name="triggerCombo">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <item>
       <property name="text">
        <string>Free</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Keyboard + Midi</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>All</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>1151</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="assignButton">
        <property name="text">
         <string>Quick assign</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>points</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="fromSpin">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>191</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>to</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="toSpin">
        <property name="maximum">
         <number>191</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>to value</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="valueSpin">
        <property name="minimum">
         <number>-128</number>
        </property>
        <property name="maximum">
         <number>127</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>(use mouse wheel)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Right click reset mode:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="resetMode">
        <item>
         <property name="text">
          <string>Original value</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Null value</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DialKnob</class>
   <extends>QDial</extends>
   <header>View/UiWidgets.h</header>
  </customwidget>
  <customwidget>
   <class>ComboPicker</class>
   <extends>QComboBox</extends>
   <header>View/UiWidgets.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
